docusaurus/docs/Android/01-basics/05-core-concepts.mdx

```diff
@@ -79,50 +79,68 @@ viewModelScope.launch {
 
 Actions defined in a `Call` return `Result` objects. These contain either the result of a successful operation or the error that caused the operation to fail.
 
-You can check whether a `Result` is successful or an error - exactly one of the following will be true for each `Result`:
+If the result was successful, you can get the contained data. Otherwise, you can fetch the error and handle it appropriately.
```

Comment:
Grammarly suggestions:

```suggestion
If the result is successful, you can get the contained data. Otherwise, you can fetch the error and handle it appropriately.
```